test/versioning-strategies/always-bump-minor.ts

```diff
@@ -0,0 +1,142 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {describe, it} from 'mocha';
+
+import {expect} from 'chai';
+import {AlwaysBumpMinor} from '../../src/versioning-strategies/always-bump-minor';
+import {Version} from '../../src/version';
+
+describe('MinorVersioningStrategy', () => {
```

Comment:
Tiny nit, this should be `AlwaysBumpMinorStrategy`, my bad because I copy pasted this.